monitoring/loki-config.yml

```
@@ -0,0 +1,30 @@
+
+schema_config:
+  configs:
+    - from: 2025-05-02
+      store: boltdb-shipper
+      object_store: filesystem
+      schema: v11
+      index:
+        prefix: index_
+        period: 24h
+ruler:
+  alertmanager_url: http://localhost:9093
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)